### DISCUSSION TITLE
What was the name of Long John Silver's 🦜?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#909] [Galaxy Trivia] [answered]
**Bonus:** How much does it cost a 🏴‍☠️ to get a piercing?

**Comment by peterbaccaro** [comment, marked as answer]
The name of Long John Silver's parrot, as depicted in the novel "Treasure Island" by Robert Louis Stevenson, is "**Captain Flint**". Captain Flint is a colorful and talkative parrot that often perches on Long John Silver's shoulder and plays a significant role in the story.

It costs a pirate a **buck-an-ear (buccaneer)** to get a piercing.

**Comment by raeef1001** [comment]
The name of Long John Silver's parrot, as depicted in the novel "Treasure Island" by Robert Louis Stevenson, is "Captain Flint". Captain Flint is a colorful and talkative parrot that often perches on Long John Silver's shoulder and plays a significant role in the story.

It costs a pirate a buck-an-ear (buccaneer) to get a piercing.
